src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java

```diff
@@ -0,0 +1,226 @@
+package com.wikia.webdriver.testcases.seotests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * @ownership Seo
+ */
+public class HtmlTitleTests extends NewTestTemplate {
+
+  private static final String TEST_WIKI_CUSTOM_TITLE = "es.pokemon";
+  private static final String TEST_WIKI_ORIGINAL_TITLE = "sktest123";
+
+  Credentials credentials = Configuration.getCredentials();
+
+  @DataProvider
+  private Object[][] dataHtmlTitleTest001() {
+    return new Object[][]{
+        // original title
+        {
+            TEST_WIKI_ORIGINAL_TITLE,
+            "Sktest123_Wiki",
+            "Sktest123 Wiki - Wikia",
+            "Sktest123 Wiki - Sktest123 Wiki - Wikia"
+        },
+        {
+            TEST_WIKI_ORIGINAL_TITLE,
+            "Style-5H2",
+            "Style-5H2 - Sktest123 Wiki - Wikia",
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Style-5H2 - Sktest123 Wiki - Wikia" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)